AzureTable/Microsoft.DataTransfer.AzureTable/Source/AzureTableLocationMode.cs

```diff
@@ -0,0 +1,30 @@
+﻿
```

Comment:
Unnecessary empty line. I assume a leftover from Organize usings - Remove and Sort.